unit/util/unicode.cpp

```diff
@@ -6,20 +6,22 @@ Author: Vojtech Forejt, forejtv@diffblue.com
 
 \*******************************************************************/
 
-#include <cassert>
+#include <testing-utils/catch.hpp>
+
 #include <vector>
 #include <string>
 #include <codecvt>
-#include <iomanip>
-#include <iostream>
 #include <locale>
 
 #include <util/unicode.h>
 
+// the u8 prefix is only available from VS 2015 onwards
+#if !defined(_MSC_VER) || _MSC_VER >= 1900
```

Comment:
Check with @kroening but I think this is our minimum compiler on MS environments anyhow.

Reply:
Well, at least AppVeyor's VS seems to be older than that (it does not support the `u8` prefix).

Reply:
I'll do a PR that upgrades AppVeyor to 2013. There are no stakeholders that I know of that still care about VS 2013.